rdt-open-reader/src/main/java/io/ona/rdt/activity/PatientRegisterActivity.java

```diff
@@ -54,6 +57,7 @@
 public class PatientRegisterActivity extends BaseRegisterActivity implements SyncStatusBroadcastReceiver.SyncStatusListener, OnFormSavedCallback, PatientRegisterActivityContract.View {
 
     public static final String LATEST_SYNC_DATE_FORMAT = "dd MMM hh:mm a";
+    public static final String ACTION_UPDATE_LATEST_SYNC_DATE = "action_update_latest_sync_date";
```

Comment:
could you make the value `io.ona.rdt.action_update_latest_sync_date` for global uniqueness?

Reply:
done